assets/components/radioToggle/radioToggle.jsx

```diff
@@ -40,6 +40,7 @@ export default function RadioToggle(props: PropTypes) {
     const className = getClassName(props);
     const accessibilityHintId = `accessibility-hint-${radioId}`;
     const accessibilityHint = <p id={accessibilityHintId} className="accessibility-hint">{radio.accessibilityHint}</p>;
+    /* eslint-disable jsx-a11y/label-has-for */
```

Comment:
Why is this disabled? `htmlFor` is present so there should be no lint error surely?

Reply:
That rule also enforces that `label` should wrap the `input` and this was broken. Therefore, I disabled for these lines and I created a card to fix it in the future since it was getting a huge PR.